lib/jekyll-import/importers/easyblog.rb

```diff
@@ -33,7 +33,7 @@ def self.process(options)
         user    = options.fetch("user")
         pass    = options.fetch("password", "")
         host    = options.fetch("host", "localhost")
-        section = options.fetch("section", "1")
+        # section = options.fetch("section", "1")
```

Comment:
no comments. if a line is no longer needed, let's just remove it.

Reply:
why its it no longer needed?

Reply:
Those and following were detected as unused variables.
https://rubocop.readthedocs.io/en/latest/cops_lint/#lintuselessassignment

Reply:
In doubt, I'll just exclude this file from the cop